tests/api/features/boards.feature

```diff
@@ -0,0 +1,119 @@
+        Given the user defines a "GET" request to "/boards/623cd8eef6a5ea0970e67c12/members"
+        When the user sends the request
+        Then verify the status code is "200"
+        And verify there is at least 1 member
+
+    Scenario: Updating information of a Board
+        Given the user defines a "PUT" request to "/boards/6287b012e56fca6695059571"
+            | Key         | Value                      |
+            | name        | testingnicolasv            |
+            | desc        | a valid data table         |
+        When the user sends the request
+        Then verify the status code is "200"
+        And verify the new object contains the following info
+            | Key         | Value                      |
+            | name        | testingnicolasv            |
+            | desc        | a valid data table         | 
+
+    Scenario: Updating information of a Board and verify the board
+        Given the user defines a "PUT" request to "/boards/6287b012e56fca6695059571"
+            | Key         | Value                      |
+            | name        | testingnicolasv            |
+            | desc        | a valid data table         |
+        When the user sends the request
+        Then verify the status code is "200"
+        And verify the board contains the following info
+            | Key         | Value                      |
+            | name        | testingnicolasv            |
+            | desc        | a valid data table         |
+
+    Scenario: CREATE and DELETE a board
+        Given the user defines a "POST" request to "/boards"
+            | Key         | Value                      |
+            | name        | testingnicolasv            |
+            | desc        | a valid data table         |
+        When the user sends the request
+        Then verify the status code is "200"
+        And verify the new object contains the following info
+            | Key         | Value                      |
+            | name        | testingnicolasv            |
+            | desc        | a valid data table         | 
+        And DELETE a Board
+
+      
+
+    Scenario: CREATE and DELETE a board2
+        Given the user defines a create_board function
+            | Key         | Value                      |
+            | name        |kiarajose12                 |
+           
+        When the user calls the create_board function
+        Then verify the status code is "200"
+        And verify the new object contains the following info
+            | Key         | Value                      |
+            | name        | kiarajose12            |
+        And DELETE a Board
+
+    
+    
+    Scenario: Duplicate a Board and DELETE it
+        Given the user defines a duplicate function with id "6287b86ed19bf5054179cb2f" and new name "NewName"
+        When the user sends the copy_board function
+        Then verify the status code is "200"
+        And DELETE a Board
```

Comment:
- avoid conjunctive steps (AND)
- copy_board??? 
the flow and even the code tends to be a mess when using technical vs. functional or behavioral style statements

example of technical statements
```suggestion
        Given the user defines "POST" request to "/members/me/boards"
         | Parameter        |  Value  |
         | idBoardSource  | 6287b86ed19bf5054179cb2f |
         | name                | NewName |
        When the user sends the request
        Then verify the status code is "200"
        When the user defines "GET" request to "/boards/6287b86ed19bf5054179cb2f"
        And the user sends the request
        Then verify the status code is "200"
        And verify the source board and new board data matches except by
                 | params|
                 | id  |
                 | name | 
        When the user defines "DELETE" request to "/boards/{boardID}"
        And the user sends the request
        Then verify the status code is "200"
        When the user defines "GET" request to "/boards/{boardID}"
        And the user sends the request
        Then verify the status code is "404"
```

behavioral style
``` suggestion
       Given the user created a board with following data
                | Parameter        |  Value  |
                | idBoardSource  | 6287b86ed19bf5054179cb2f |
                | name                 | NewName                  |
        When the user retrieved the source board "6287b86ed19bf5054179cb2f"
        Then verify the source board and new board data matches except by
                 | params|
                 | id  |
                 | name |                  
        When the user deletes the board "NewName"
        Then verify the board "NewName" cannot be found
```

Reply:
What do you men with "avoid conjunctive steps (AND)"  I mean, when should we use And when not? I don't catch the logic, shouldn't that be used after a When or give step?